src/translations/translations_NO.json

```diff
@@ -477,9 +477,17 @@
   },
   "work_list" : {
     "aria_label_tabs": "Faner for mine arbeidslister",
+    "comment": "Kommentar",
+    "comment_explanation" : "Skriv en kommentar hvorfor forespørselen blir avvist",
+    "create_doi": "Opprett DOI",
+    "delete_request": "Slett forespørsel",
     "doi_request_list" : "DOI forespørsler",
+    "hide": "Skjul",
     "page_title" : "Arbeidsliste",
     "page_requirements": "Du må være kurator eller redaktør for å få tilgang til denne funksjonaliteten",
-    "reports" : "Rapporteringer"
+    "read_more": "Les mer",
```

Comment:
finnes det ikke en ferdig komponent man kan bruke for denne funksjonaliteten? Ellers var det helt grei løsning altså. Trenger vi det flere plasser kan vi kanskje trekke ut logikken

Reply:
Gjorde litt research, det eneste jeg fant var denne artikkelen, og med dine forslag til bedre navngivning på knappen så tror jeg ikke vi trenger aria-label for å vise frem resten.
https://jolvera.dev/posts/accessible-read-more-links

Reply:
super